src/providers/github/users.js

```diff
@@ -38,12 +38,24 @@ async function getUser(token) {
       'user-agent': 'cool-bio-analytics-github-oauth-login',
     };
     console.log('[user getUser headers]', headers);
-    const getUserResponse = await fetch('https://api.github.com/user', {
+      const getUserResponse = await fetch('https://api.github.com/user', {
       method: 'GET',
       headers,
     });
     const data = await getUserResponse.json();
     console.log('[provider user data]', data);
+    if (!data.email) {
+          // If the user does not have a public email, get another via the GitHub API
+          // See https://docs.github.com/en/rest/users/emails#list-public-email-addresses-for-the-authenticated-user
+          const res = await fetch("https://api.github.com/user/emails", {
+                method: 'GET',
+                headers,
+          });
+          const emails: GithubEmail[] = await res.json()
```

Comment:
This will throw an error as `GithubEmail` is not defined? I am not using typescript ATM.

Reply:
you're right - sorry, I was testing in a different TS app ... I just removed the type now